### DISCUSSION TITLE
Changing Pro subscription

### DISCUSSION THREAD

**Discussion by candroid-man** [streetwriters/notesnook#719] [Ideas]
### What problem are you facing?

I feel like the Basic version of Notesnook is a little too limiting. The most basic of things are locked behind a paywall, such as full editor and unlimited notebooks/tags, etc. This could easily prevent users from choosing Notesnook over a different option. Besides, once Notesnook is open source, a user could easily fork Notesnook and remove this paywall.

### What's the solution you'd like to see in Notesnook?

I think the following should be removed from the Pro version, and added to the Basic plan:

- Vault (locked notes)
- Exporting
- Full rich editor
- Unlimited Notebooks and Tags

If you think this is extreme, most privacy respecting notes apps only charge for cloud storage. That is the current norm, making this a clearly inferior option compared to something like [Cryptee](https://crypt.ee/#pricing). People would gladly pay $4.50 a month or $50 a year for unlimited cloud storage.

### What alternatives have you considered?

To encourage more people to buy Pro, you could alternatively make sync a paid feature rather than unlimited storage (since everything would be stored locally and not stored in the cloud). Paying for cloud storage is a much more reasonable than paying for basic features and makes more sense overall to potential customers.

**Comment by thecodrr** [comment]
> Besides, once Notesnook is open source, a user could easily fork Notesnook and remove this paywall.

Unfortunately, that's one of the cons of going open source. But that's open source and nothing we can do about it.

> I think the following should be removed from the Pro version, and added to the Basic plan:
> 1. Vault (locked notes)
> 2. Exporting
> 3. Full rich editor
> 4. Unlimited Notebooks and Tags
> If you think this is extreme, most privacy respecting notes apps only charge for cloud storage.

To be honest, there aren't that many private note taking apps of considerable reputation to draw a comparison. Take Standard Notes for example: it locks away all the editors behind a paywall (much more expensive than Notesnook). Cryptee on the other hands limits storage but gives away all features for free.

Which is better? If you look at the pricing chart of Notesnook you'll notice that we try to strike a good balance between features & cloud sync.

The main problem with going one way or the other is that there will always be someone who will want everything they use for free — and they might even be right from their perspective. If you use the Basic plan of Notesnook, it doesn't hinder you at all if you only take basic kind of notes. However, as soon as you start venturing into more complex features (notebooks, tasks & lists in editor, tags etc.) Notesnook starts to close down. It doesn't make a sound if you are syncing thousands of notes between multiple devices. A lot of people might prefer that to paying for storage or syncing.

> People would gladly pay $4.50 a month or $50 a year for unlimited cloud storage.

I am not sure about this. If this is true, the other could be true as well right? That people would gladly pay $4.49/mo to unlock the Pro plan. It's certainly not a lot of money for the features we are offering (compare that to Standard Notes which is priced at ~$9/mo)

> To encourage more people to buy Pro, you could alternatively make sync a paid feature rather than unlimited storage

With all that being said, it is in our plans to create a flexible Pro plan that users can add/remove features from themselves. In essence, if someone just wants the full rich editor they can choose to only pay for that (obviously, it'll much less expensive). Or they can pay for rich editor + organization etc.

**In conclusion:**

Almost everything private (except the vault) is available for free. That is one of our primary goals with Notesnook; to make privacy the default. That is why we have to choose other features to put behind a paywall.

**Reply by thecodrr** [reply]
Here's a simple comparison between Cryptee & Notesnook pricing:

| Storage | Cryptee  | Notesnook (notes + attachments) | Notesnook (only notes)
| --- | --- | --- | -- |
| 100 MB | free | $4.49/mo | free |
| 10  GB | $3/mo | $4.49/mo | free |
| 400 GB | $9/mo | $4.49/mo | free |
| 2 TB | $27/mo | $4.49/mo | free |
| Unlimited | - | $4.49/mo | free |

Which looks better?

**Comment by candroid-man** [comment]
> Unfortunately, that's one of the cons of going open source. But that's open source and nothing we can do about it.

Well, you can't fork the app to get free storage from YOUR servers. My suggestion would be a way for you to make sure that this can't happen

> I am not sure about this. If this is true, the other could be true as well right? That people would gladly pay $4.49/mo to unlock the Pro plan. It's certainly not a lot of money for the features we are offering (compare that to Standard Notes which is priced at ~$9/mo)

Personally, I feel that it is more in the spirit of freedom of software to charge for storage because it is more understandable as to why you are charging for the product. The paywall for actual features of the app just feels more like what a proprietary app would do, and I feel the same way about Standard Notes doing it this way as well. Another personal note: I would definitely prefer having all the features in the free version and pay for unlimited storage/sync, and I think more people would use the app locally. I feel organization is a key feature of a notes app and limiting that almost forces people that use it a lot to buy Pro.

It also really gives the user more choice from a privacy and security standpoint, in the sense that not everyone wants their data (encrypted or not) in a server, or anywhere besides their devices' storage. Changing the plan to what I am suggesting would allow users to have that choice, whether they want to have a local notes app with a great editor, or an online notes app with a great editor.

You are right though, some people may prefer it the way it is. I think this is a good time to get a vote from the users to see what everyone would prefer the most.

> Here's a simple comparison between Cryptee & Notesnook pricing

Keep in mind that Cryptee is not only a notes app, it is also a document drive and photo gallery, and they are not offering unlimited storage. It makes more sense for them to structure their plans this way because larger files are involved. If they offered unlimited storage, that could be easily abused and cause problems for the small company (which I'm sure you can relate to).

**Comment by at-xa** [comment]
I think if you want to incentivize people paying for a service at a price point that you feel as a developer you can't go any lower than, you should implement and extend the free trial strategy. The initial concern I have as user is always price, and features comparison. It isn't that I'm unwilling to pay a for service that's good, it's that my brain immediately goes to why do I have to pay at all if I don't know it's good. I think you should consider giving users something like 3 months free trial or so, with all pro features, and really sell this point as much as you can, like with a youtube video with a rundown of everything this gives you. I wonder how many users you can convert to a paying method this way because it gives them a chance to make the service a part of their lives and then they can judge from its usefulness whether they really need it or not, instead of whether they want it or not.

**Reply by thecodrr** [reply]
Not sure if you have checked but there's already a 14 day free trial.

**Reply by at-xa** [reply]
- Yes, I suppose I meant extend the trial period when I said implement. It's sort of like saying, the pressure for a user to commit to the service is removed, and (hopefully) the usefulness of the service is adequately measured, because the user has spent an extended amount of time with the service's full feature-set... I, as a user, would say.. (Wow) these featues look cool -- (Wow) I get three months of every feature before I have to decide to commit financially... etc.. This kind of goodwill strategy of simply extending the free trial period could help differentiate you from other privacy note apps.

**Reply by thecodrr** [reply]
You raise a good point and put forward a good case. However, the main question is: is a 14-day trial not enough to evaluate a product? If it isn't then what would be an adequate amount? 3 months? By what measure? What if even 3 months aren't enough?

**Reply by at-xa** [reply]
I actually don't know what the sweet spot is.. what your costs are.. you'd also have take into account how well-established your features are compared to other apps.. because you'd absolutely want to maximize your app's impression value.. etc.. But if you're ever in a position where you're not satisfied with your user-count and you're not willing to lower prices.. This is a promotion strategy I haven't seen before.. (I think.)

Best of luck.

**Comment by suny24x7** [comment]
A lot of good discussions are going on. I would like to add something, although I am not a software developer.

Since Notesnook is going open source, I don't think it would be good to have a paywall on the client side. What Notesnook can do instead is offer a 1/2/3 month free trial that includes all pro features. Once the free trial ends, users will only be able to enjoy all the features of the app but will not get any backup or sync services.

Nootsnook will offer two paid models to users.

1. One time payment.

2. Monthly subscription.

The one-time payment can be promoted as if it supports the project. In return, users will get unlimited cloud backup. This will be their gift for their support. This one time payment will not include instant sync facility, only cloud backup. (Notesnook may offer 10/20GB if they feel it is not possible to offer unlimited backup with a one-time payment.) And free users should be allowed to handle their own data as Obsidian does. 

And monthly subscribers will continue to get the benefits they currently have.

**Reply by rufusgerm** [reply]
> The one-time payment can be promoted as if it supports the project. In return, users will get unlimited cloud backup.

What does this mean exactly? Where are these backups going?

**Comment by trymeouteh** [comment]
I would like to make a suggestion to the plans and what each plan offers. Essentially users get all of the features for free but with limited storage. Plans will just be for more storage space on the cloud.

1. Please consider having the free basic plan have all the features that are available in the pro plan but with limited storage (Such as 50MB of storage). This way those with free accounts get all the features, especially when sharing notes with others but to get a good amount of storage, they can pay for a plan.
2. More plan options. Have cheaper plans with less storage such as a 1GB storage plan, 5GB storage plan, etc. And the current Pro plan could be changed to have a limit on storage space such as 5GB.

I believe this is a more FOSS way of offering a service since all features are free, but if you want more storage space on a server, you got to pay for renting that storage space.
